MatrixSDK/Crypto/MXCrypto.m

```diff
@@ -933,6 +934,100 @@ - (void)setDevicesKnown:(MXUsersDevicesMap<MXDeviceInfo *> *)devices complete:(v
 #endif
 }
 
+- (void)setUserVerification:(BOOL)verificationStatus forUser:(NSString*)userId
```

Comment:
We can maybe use `- (void)setUserWithId:(NSString*)userId verified:(BOOL)isVerified` for readability.